application/clicommands/ProcessCommand.php

```diff
@@ -150,15 +160,34 @@ protected function renderProblemTree($tree, $useColors = false, $depth = 0)
             if ($useColors) {
                 $state = $this->screen->colorize($state, $colors[0], $colors[1]);
             }
-
-            $output .= sprintf(
-                "%s%s %s %s\n",
-                str_repeat('  ', $depth),
-                $node instanceof BpNode ? $node->getOperator() : '-',
-                $state,
-                $node->getAlias()
-            );
-            $output .= $this->renderProblemTree($subtree['children'], $useColors, $depth + 1);
+            if ($htmlcolors) {
+                $state = substr($state, 1, strlen($state)-2);
```

Comment:
Is this just for undoing `colorize()`? If so, why don't you just avoid `colorize()` in this case?